internal/host/validator.go

```diff
@@ -700,16 +700,17 @@ func (v *validator) isIgnitionDownloadable(c *validationContext) ValidationStatu
 	if c.infraEnv != nil {
 		return ValidationSuccessSuppressOutput
 	}
-	if c.inventory == nil {
-		return ValidationPending
-	}
 	if !hostutil.IsDay2Host(c.host) || swag.BoolValue(c.cluster.UserManagedNetworking) {
 		return ValidationSuccessSuppressOutput
 	}
+	if c.host.APIVipConnectivity == "" {
```

Comment:
Can you please add tests?

Reply:
Sorry missed the earlier comment! Will do!